### ISSUE TITLE
`unrecognized-char-ref` rule reported false positives

### ISSUE THREAD

**Issue by Chocobo1** [html-validate/html-validate#15]
`unrecognized-char-ref` started reporting false positives in `<script>` after upgrading to `html-validate@10.1.0`.

## Configuration

Using `html-validate:recommended`.

`test.html`:
```html
<script>
    const url = `test.html?path1=abc&path2=abc`;
</script>
```

Command-line: `npx html-validate test.html`

Result:
```
/home/test/test.html
  1:46  error  Unrecognized character reference "&path2"  unrecognized-char-ref

✖ 1 problem (1 error, 0 warnings)

More information:
  https://html-validate.org/rules/unrecognized-char-ref.html
```

## Expected result

No error.

## Version

- `html-validate`: 10.1.0

**Comment by ext** [comment]
Thank you for reporting this, there is a bugfix on the way.
